includes/admin/reporting/class-sales-logs-list-table.php

```diff
@@ -52,6 +52,18 @@ public function __construct() {
 	}
 
 	/**
+	 * Gets the name of the default primary column.
+	 *
+	 * @since 2.5
+	 * @access protected
+	 *
+	 * @return string Name of the default primary column.
+	 */
+	protected function get_default_primary_column_name() {
+		return 'ID';
```

Comment:
@pippinsplugins I agree. I originally tried that but it looks like this:

![image](https://cloud.githubusercontent.com/assets/1254870/10258958/03fb6344-6931-11e5-994b-8551f05ea27f.png)

Same for some of the other ones. Thoughts about that? Seems like any 'primary' column that isn't the first one shows up weird like that. Am I missing something?